homeassistant/components/ipma/weather.py

```diff
@@ -331,10 +332,10 @@ def forecast(self):
                         ),
                         None,
                     ),
-                    ATTR_FORECAST_TEMP_LOW: data_in.min_temperature,
-                    ATTR_FORECAST_TEMP: data_in.max_temperature,
+                    ATTR_FORECAST_NATIVE_TEMP_LOW: data_in.min_temperature,
+                    ATTR_FORECAST_NATIVE_TEMP: data_in.max_temperature,
                     ATTR_FORECAST_PRECIPITATION_PROBABILITY: data_in.precipitation_probability,
-                    ATTR_FORECAST_WIND_SPEED: data_in.wind_strength,
+                    ATTR_FORECAST_NATIVE_WIND_SPEED: data_in.wind_strength,
```

Comment:
wind_strength is a class and not a name: https://api.ipma.pt/open-data/wind-speed-daily-classe.json

Reply:
In that case, it should not be used at all, as wind speed should be a numeric value.

Reply:
That will be a breaking change, the value is numeric (wind strength) which is converted to a string according to the doc in my comment.

If it must be a number I will leave the number from the API, but it will be a regression in UX

Reply:
Windspeed must be a `float`, it always has been that way (and documented as such in both typing and the developer documentation). The accepted units previous have been km/h or mi/h. With the new logic (being applied here), we can handle more units: m/s, km/h, mi/h, ft/s, or kn.

Reply:
This very weird, the ipma tests have wind speed forecasts in km/h, not a descriptive text in Portuguese or English.

Reply:
wind forecast is broken... I'm fixing this in another PR